pkg/apis/app/v1alpha1/kogitodataindex_types.go

```diff
@@ -89,11 +89,11 @@ type KogitoDataIndexStatus struct {
 	// Add custom validation using kubebuilder tags: https://book-v1.book.kubebuilder.io/beyond_basics/generating_crd.html
 
 	// Status of the Data Index Service Deployment created and managed by it
-	// +operator-sdk:gen-csv:customresourcedefinitions.statusDescriptors=true
+	// +operator-sdk:gen-csv:customresourcedefinitions.statusDescriptors=false
```

Comment:
just for clarification, what is the purpose ?

Reply:
The `DeploymentSpec.Status` attribute doesn't play well with the CSV status display page, it's not rendered and the OCP console raises a NodeJS error. I'll change this status spec in the Service Definition Framework I'm working on.

Reply:
FYI: https://issues.redhat.com/browse/KOGITO-853